package.json

```diff
@@ -41,8 +42,10 @@
     "js-yaml": "^3.14.1",
     "lodash.throttle": "^4.0.8",
     "panini": "^1.7.1",
+    "qrcode": "^1.4.4",
```

Comment:
Please check for licensing, MIT is good

Reply:
MIT

Source:
https://www.npmjs.com/package/qrcode